RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerMultiFit.h

```diff
@@ -94,7 +95,7 @@ class EcalUncalibRecHitWorkerMultiFit final : public EcalUncalibRecHitWorkerBase
   edm::ESGetToken<EcalSampleMask, EcalSampleMaskRcd> sampleMaskToken_;
 
   // time algorithm to be used to set the jitter and its uncertainty
-  enum TimeAlgo { noMethod, ratioMethod, weightsMethod };
+  enum TimeAlgo { noMethod, ratioMethod, weightsMethod, kansasMethodCC };
```

Comment:
Maybe a more descriptive name than kansasMethodCC could be found?

Reply:
crossCorrelationMethod?

Reply:
Sounds OK to me.